src/parser/Tokenizer.h

```diff
@@ -133,31 +124,32 @@ struct TurtleTokenCtre {
   static constexpr auto ExponentString = fixed_string("[eE][\\+\\-]?[0-9]+");
   static constexpr auto Exponent = grp(ExponentString);
 
-  static constexpr auto DoubleString = fs("[\\+\\-]?([0-9]+\\.[0-9]*") +
-                                       ExponentString + "|" + ExponentString +
-                                       ")";
+  static constexpr auto DoubleString =
+      fixed_string("[\\+\\-]?([0-9]+\\.[0-9]*") + ExponentString + "|" +
+      ExponentString + ")";
```

Comment:
I appreciate the explicit `fixed_string` instead of the abbreviation `fs`.

Is `fs` still used elsewhere?

Reply:
the `fs` is not used, and already is deleted in the code you are looking at.